src/main/java/net/wasdev/wlp/ant/AbstractTask.java

```diff
@@ -352,12 +352,13 @@ protected String findStringInFile(String regexp, File fileToSearch) throws Excep
      *            a regular expression (or just a text snippet) to search for
      * @param fileToSearch
      *            the file to search
+     * @param msgLevel the log level for the match number message
      * @return List of Strings which match the pattern. No match results in an
      *         empty list.
      * @throws Exception
      */
-    protected List<String> findStringsInFileCommon(String regexp,
-                                                   boolean stopOnFirst, int searchLimit, File fileToSearch)
+    private List<String> findStringsInFileCommon(String regexp,
+                                                   boolean stopOnFirst, int searchLimit, File fileToSearch, int msgLevel)
```

Comment:
Do we need to have this method throw `Exception`? Everything just looks like it's in a big try/catch block. If we can get rid of the exception being thrown here and in the other overloaded method then we could also eliminate the try/catch starting from line 487.

Reply:
done